src/components/RightSidebar/SharedItems/SharedItems.vue

```diff
@@ -15,6 +15,10 @@
 				<LocationCard wide v-bind="item.messageParameters.object" />
 			</div>
 
+			<PinnedMessageItem
+				v-else-if="isPinned"
+				:pinned-message="item" />
```

Comment:
It should be pretty clear inside of PinnedMessageItem, what the item is

```suggestion
				:item />
```